test/testFunction.php

```diff
@@ -0,0 +1,38 @@
+<?php
+require_once '../function.php';
+
+use PHPUnit\Framework\TestCase;
+
+class FunctionTest extends TestCase
+{
+    public function testGenerateRequestShouldReturnEverythingIfNoParameter()
+    {
+        //setup
```

Comment:
It was setup, execution and assertion but I only added those comments to demonstrate the stages of a test, you don't need these comments in the tests

Reply:
ok removed